tests/abi/test_pkg_removal.sh

```diff
@@ -13,18 +13,21 @@ if [ "$(dpkg --print-architecture)" != "amd64" ]; then
     exit 0
 fi
 
-diff -u \
-     <(tail -n +6 prime/usr/share/snappy/dpkg.list|awk '{print $2}'|sort) \
-     <(cat <<EOF
+DIFF=$(comm -1 -3 \
+         <(tail -n +6 prime/usr/share/snappy/dpkg.list|awk '{print $2}'|sort) \
+         <(cat <<EOF
 adduser
+apparmor
```

Comment:
Now that we only look at removals we can remove the additions to the list here, right?

Reply:
I guess we can, I left them around since I thought the list needed an update anyway. But we might just leave the original list and only add those things that we care about indeed. Should I proceed?